sdk/documentintelligence/Azure.AI.DocumentIntelligence/samples/Sample_AddOnCapabilities.md

```diff
@@ -0,0 +1,398 @@
+# Analyze a document with add-on capabilities
+
+This sample demonstrates how to analyze document with add-on capabilities, using a document as an example. For more information about add-on capabilities, see the [documentation][docint_addon].
+
+To get started you'll need a Cognitive Services resource or a Document Intelligence resource. See [README][README] for prerequisites and instructions.
+
+Add-on capabilities are available within all models except for the Business card model. This sample uses Layout model ("prebuilt-layout") to demonstrate. When analyzing a document, you can specify a list of optional [analysis features][sdk_docfeature] to enable certain add-on capabilities.
+
+The following capabilities are free:
+- Barcodes
+- Languages
+- KeyValuePairs
+
+The following capabilities will incur additional charges, see [pricing][docint_pricing]:
+- Formulas
+- OcrHighResolution
+- FontStyling

```

Comment:
We usually avoid this kind of information (pricing, supported models) in the SDK docs since they tend to get outdated pretty quickly. It's preferable to link to service documentation so the information gets centralized in one place.

Reply:
Sure. Will remove them.